.devcontainer/welcome.txt

```diff
@@ -0,0 +1,9 @@
+~~~~~~ Welcome to vets-api on codespaces! ~~~~~~
```

Comment:
Do we need a banner for every terminal the user opens?

Reply:
It will only show the first time a terminal is opened.